lib/game.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'frame'
+
+class Game
+  attr_reader :frame, :scores
+
+  TP = 10
+  TF = 10
+  def initialize
+    @scores = []
+    @frame = 0
+  end
+
+  def roll(pins)
+    raise 'Your game is now complete' if game_over?
+
+    @scores << pins
+  end
+
+  def score
+    result = 0
+    index = 0
+    TF.times do
+      @frame += 1
```

Comment:
Can you use a local variable to iterate over frames here?

Reply:
More generally, instance variables should be storing things that say something about the instance, and need to live for as long as the instance lives. They are not just shared memory between methods. If you want to pass state to a different method called by a first method, you should use an argument.